### PR TITLE
issue #42 : raise exception in zdt1 if size <= 1

### PR DESCRIPTION
**Pull request by geoffreyp**

### CODE REVIEW

File: moead_framework/problem/numerical/zdt.py [hunk 1]
```diff
@@ -20,6 +20,10 @@ def __init__(self, size):
         :param size: {integer} number of variables in a solution
         """
         self.n = size
+
+        if self.n <= 1:
```
Comment (chkoar) [inline moead_framework/problem/numerical/zdt.py:24]: `n` should be a positive integer right? What message will I get if I pass a string or float?

**Comment by geoffreyp** [comment]
raise new exception in eff65dc : 

```
        if type(self.n) != int:
            raise ValueError("The parameter 'size' must be an integer strictly greater than 1 (size > 1)")
```

File: moead_framework/problem/numerical/zdt.py (hunk 1 above)
Comment (geoffreyp) [reply moead_framework/problem/numerical/zdt.py:27]: new message in  eff65dc

Comment (chkoar) [reply moead_framework/problem/numerical/zdt.py:27]: There is not reason to duplicate the same line for a single validation.

**Comment by chkoar** [comment]
I think you should rebase, merge and close the issue #42

**Pull request merged**